src/node_report.cc

```diff
@@ -638,13 +638,8 @@ static void PrintResourceUsage(JSONWriter* writer) {
     writer->json_keyvalue("constrained_memory", constrained_memory);
   }
 
-  // See GuessMemoryAvailableToTheProcess
```

Comment:
Would you mind updating `GuessMemoryAvailableToTheProcess` in env.cc as well?

Reply:
Done. Please help review.